platform/elrond/mocks/tx_negative_value.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
could you rename file to scr_negative_value or something similar

Reply:
done.